extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -525,14 +528,20 @@ export class AmpStoryPage extends AMP.BaseElement {
           if (state.vh === 0 && state.vw === 0) {
             return;
           }
-          this.win.document.documentElement.setAttribute(
-            'style',
+          if (!this.cssVariablesStyleEl_) {
+            const doc = this.win.document;
+            this.cssVariablesStyleEl_ = doc.createElement('style');
+            this.cssVariablesStyleEl_.setAttribute('type', 'text/css');
+            doc.head.appendChild(this.cssVariablesStyleEl_);
+          }
+          this.cssVariablesStyleEl_.textContent =
+            `html {` +
```

Comment:
Are you just assuming documents only have one `<html>` element? 😂

It needs to be the outermost one, I think, for `rem` units to work (which, this would target all `<html>` elements in the document).  Can we use `:root`?

Reply:
Done!

Reply:
It's causing the `amp-story-auto-ads-fullbleed` tests to fail in a viewer. I will investigate further later.

Reply:
(only when minified...)